packages/fiori/src/NotificationListItem.ts

```diff
@@ -40,6 +40,12 @@ import NotificationListItemTemplate from "./generated/templates/NotificationList
 // Styles
 import NotificationListItemCss from "./generated/themes/NotificationListItem.css.js";
 
+type NotificationListItemPressEventDetail = {
+	item: NotificationListItem;
```

Comment:
I think entries in types are ending with comma ",", I might be wrong
` item: NotificationListItem,`